frontend/ui/uimanager.lua

```diff
@@ -1113,26 +1116,32 @@ function UIManager:sendEvent(event)
         end
     end
 
-    -- if the event is not consumed, active widgets (from top to bottom) can access it.
-    -- NOTE: _window_stack can shrink when widgets are closed (CloseWidget & Close events).
+    -- If the event was not consumed (no handler returned true), active widgets (from top to bottom) can access it.
+    -- NOTE: _window_stack can shrink/grow when widgets are closed (CloseWidget & Close events) or opened.
+    --       Simply looping in reverse would only cover the list shrinking, and that only by a *single* element,
+    --       something we can't really guarantee, hence the nuclear option below.
```

Comment:
"nuclear option" ? You mean "complex code", or some "atomic parameter" that I don't see ? :)

Reply:
It's just a mildly uncommon iteration method that's slightly more dogged than usual ;).